app/assets/scripts/views/project/dashboard.js

```diff
@@ -46,8 +47,28 @@ function Dashboard({
           locationId={locationIds.length === 1 ? locationIds[0] : null}
           locationIds={locationIds}
           bbox={bbox}
-          firstUpdated={projectDates.start}
-          lastUpdated={projectDates.end}
+          dateRange={
+            dateRange
```

Comment:
the complexity here made me wonder if we should store the dateRange in state as {start, end} directly instead? I did not want to do all the related refactoring at this point, but it would maybe be a good idea?